### PR TITLE
Create creating-billing-plan.liquid

### PR DESCRIPTION
**Pull request by diana-lakatos**
Creating a Billing Plan tutorial for the Billing System Launch.

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/partner-portal/creating-billing-plan.liquid [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+converter: markdown
+metadata:
+  title: Creating a Billing Plan
+  description: This guide will help you create your own Billing Plan.
+slug: tutorials/partner-portal/creating-billing-plan
+searchable: true
+---
+
+This guide will help you create your own Billing Plan, that you can then assign to your SubPartners (clients). Your Billing Plan will be a combination of an already existing Billing Plan available to you, and the
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/partner-portal/creating-billing-plan.liquid]: and the..? :)

File: marketplace_builder/views/pages/tutorials/partner-portal/creating-billing-plan.liquid [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+### Step 2: Create Billing Plan
+
+Click on `Billing Plans`. A list of previously created Billing Plans is displayed (or none, if you haven't created any Billing Plans yet). Click on `Create Billing Plan`, and fill in the form.
+
+* **Parent Plan Type**: Select one of the Billing Plans available to you. The Parent Billing Plan will be the basis of the Billing Plan you're building. For each Instance created with your newly created Billing Plan, you will be charged according to its Parent Plan Type. 
+
+{% capture content %}
+Currently, you can see a "Staging Unbilled" Parent Plan Type in the list. Staging Instances will remain Unbilled for now, we will create a separate plan for staging, dev, and QA Instances later. 
+
+Billing Plans not marked "Unbilled" work with a flat hosting fee without overages (overages will be introduced later and invoiced retroactively).
```
Comment (kv109) [inline marketplace_builder/views/pages/tutorials/partner-portal/creating-billing-plan.liquid]: remove 
> not marked "Unbilled"

to make it simpler. Unbilled also works with flat fee, there is just no fee ;)

**Review by kv109 — APPROVED** [review]

**Review by kv109 — APPROVED** [review]

File: marketplace_builder/views/pages/tutorials/partner-portal/creating-billing-plan.liquid [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+### Step 3: Make Billing Plan available
+
+Now, you have to make the Billing Plan you created available to select when you want to assign them to your SubPartner's Instances. 
+
+* **Select Partners which can use [Billing Plan name]**: In the drop-down, select SubPartners you'd like to share this plan with. Then click on `Grant permissions`. 
+* **Select Environments**: Select which environments this Billing Plan will be available for. Then click on `Assign environments`. 
+
+## Next steps
+Congratulations! You have created your own Billing Plan. Now you can assign it to your clients' Instances.
```
Comment (lemingos) [inline marketplace_builder/views/pages/tutorials/partner-portal/creating-billing-plan.liquid]:
```suggestion
Congratulations! You have created your own Billing Plan. Now you can assign it to your clients Instances. 
```

**Review by lemingos — CHANGES_REQUESTED** [review]
just one change request

**Review by lemingos — APPROVED** [review]

**Pull request merged**